src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.PureComponent {
```

Comment:
PureComponent is a class version of React.memo. No need for it here
```suggestion
export class App extends React.Component {
```